tests/suites/os/tests/config-json/index.js

```diff
@@ -385,11 +385,11 @@ module.exports = {
             )
         );
         test.comment(`Device should show previous boot records, showed ${testcount} boots`);
-        /*test.is(
+        test.is(
           testcount > bootCount,
```

Comment:
Maybe we should try putting this back to `testcount === bootCount+2` - as technically there has been 2 boots since we measured bootCount.

Reply:
I agree. No matter whichever place the test was located at in the order. It would have a

Bootcount of x and with 2 more reboots it should have x+2 which does seem the right way to go about it.